### DISCUSSION TITLE
RMarkdown CV

### DISCUSSION THREAD

**Discussion by mjkunta** [BryanJenksCommunity/FAQ#80] [Q&A] [answered]
Hello Bryan,
First, thanks for the CV tutorial. As a beginner in R, stuff like that fires up my desire to keep learning.
My question was : let's say you've an about section and you don't want to include it in the yaml header but in the body. I know you can use ## but is there a way you can include it in your data.r file as a string and pull it from there as you did with the tibbles?

Regards,
Josiah

**Comment by tallguyjenks** [comment, marked as answer]
You can definitely declare a variable to hold a string and input it into the document

in `data.r`:

```r
my_string <- "my string"
```

in `CV.Rmd`:

```md

## Heading

content

`r my_string`

other content
```

**Reply by mjkunta** [reply]
Thank you Bryan. 
This was very helpful, I appreciate.
